tests/YearWeekTest.php

```diff
@@ -395,4 +399,31 @@ public function testToString(int $year, int $week, string $expected)
         $yearWeek = YearWeek::of($year, $week);
         $this->assertSame($expected, (string) $yearWeek);
     }
+
+    public function testNow()
+    {
+        $now = new FixedClock(Instant::of(2000000000));
+        $timeZone = TimeZone::parse('Asia/Taipei');
+        $yearWeek = YearWeek::now($timeZone, $now);
+
+        $this->assertInstanceOf(YearWeek::class, $yearWeek);
```

Comment:
Please check the (string) representation of $yearWeek, now that you've set a fixed clock!